testData/solutions/win32.json5

```diff
@@ -1535,154 +1594,177 @@
-                            ]
+
+                    // -------------- --------------------------- ----------------
+
+                    "Options.SpellingAttributeOverride": {
+                        // NOTE-TODO: Unknown setting format, but when it's selected
+                        // in the menu "Detect Spelling While Navigating" the
+                        // value is set to "|messageVoice|BuzzerShort.wav|messageVoice|"
+                        "schema": {
+                            "title": "Spelling Attribute Override",
+                            "description": "Overrides the behavior when a spelling mistake is found. Complete functionality is unknown.",
+                            "type": "string"
                         }
                     },
-                    "Options.Repetitions": {
+                    "Options.IndicateMistypedWord": {
                         "schema": {
-                            "title": "Repetitions",
-                            "description": "Whether or not to indicate repeated characters.",
-                            "type": "number",
-                            "default": 0,
-                            "enum": [
-                                0,
-                                1
-                            ],
-                            "enumLabels": [
-                                "off",
-                                "on"
-                            ]
+                            "title": "Indicate Mistyped Words",
+                            "description": "Whether to indicate a mistyped word.",
+                            "enum": [0, 1],
+                            "enumLabels": ["off", "on"],
+                            "default": 1
                         }
                     },
-                    "Options.SayAllIgnoreShiftKeys": {
+                    "Options.GrammarAttributeOverride": {
+                        // NOTE-TODO: The format of the settign is unknown, but
+                        // when the setting is activated is set to: "1|messageVoice||messageVoice|"
                         "schema": {
-                            "title": "\"Say All\" Ignores Shift Keys",
-                            "description": "Allows sticky keys to be used, i.e.: ignores shift keys during SayAll. User can use left/right arrow to do same functions.",
-                            "type": "number",
-                            "default": 0,
-                            "enum": [
-                                0,
-                                1
-                            ],
-                            "enumLabels": [
-                                "on",
-                                "off"
-                            ]
+                            "title": "Grammar and Advanced Proofing",
+                            "description": "Enable or disable 'Grammar and Advanced Proofing' feature.",
+                            "type": "string"
```

Comment:
This is a stealth variation on the pipe-delimited string pattern.  At a bare minimum the default should at least be there as a single safe example of what's actually expected.

Reply:
commented out